src/database.test.ts

```diff
@@ -0,0 +1,20 @@
+import DATABASE from "./database.json";
+
+test("a database entry of type github should have a owner and repo", () => {
+  for (const key in DATABASE) {
+    const entry = DATABASE[key];
+    if (entry.type == "github") {
+      expect(entry.owner).toBeTruthy();
+      expect(entry.repo).toBeTruthy();
+    }
+  }
+});
+
+test("a database entry should never have a path starting with /", () => {
+  for (const key in DATABASE) {
+    const entry = DATABASE[key];
+    if (entry.path) {
+      expect(entry.path).not.toMatch(/^\/.*/);
```

Comment:
Nit: it seems more readable to just check the first character than use a regular expression

Reply:
I agree - Jest does not have that built in though. I could rewrite it as `expect("entry.path".startsWith("/")).toBeFalsy();` or we could get this matcher and a lot more matcher by using the [jest-extended](https://github.com/jest-community/jest-extended) library. What do you prefer?

Reply:
expect(entry.path[0]).toBe("/")

Reply:
That works too :-)